easybuild/tools/filetools.py

```diff
@@ -1158,3 +1090,39 @@ def det_size(path):
         _log.warn("Could not determine install size: %s" % err)
 
     return installsize
+
+
+def copy_files(src, dst, symlinks=False, ignore=None):
+    """
+    Wrapper around shutil.copytree to nicely handle extended dry run
+    Arguments are the same as for the original function
+    The destination directoy must not exist
+    """
+    if build_option('extended_dry_run'):
+        dry_run_msg("Copying directory %s to %s" % (src, dst), silent=build_option('silent'))
+        return
+
+    if os.path.exists(dst):
+        raise EasyBuildError("Destination directory %s of copy already exists.", dst)
```

Comment:
I don't see why we shouldn't, if the directory is empty, that is.
Otherwise the caller is simply going to do exactly that anyway?